nce/NCE-Pairwise-SM/evaluate.py

```diff
@@ -0,0 +1,23 @@
+import shlex
+import subprocess
+
+def evaluate(instances, valid, config):
```

Comment:
You don't need this anymore, import `utils.relevancy_metrics.get_map_mrr` and use that.